umbral-pre-wasm/patches/api-compatibility.js.txt

```diff
@@ -0,0 +1,40 @@
+
+/*
+    contents added by umbral-pre-wasm makefile/build script
+*/
+
+import { KeyFrag } from  "./umbral_pre_wasm_bg.js";
+
+export const decrypt_reencrypted = (
+    receiving_sk,//: SecretKey,
```

Comment:
I guess the comments can be removed, or at least cleaned up a little

Reply:
👍